js/src/atk-utils.js

```diff
@@ -69,4 +68,37 @@ const atkUtils = (function () {
     };
 }());
 
-export { atkOptions, atkEventBus, atkUtils };
+function atkDebounce(func, wait, options) {
+    let timerId;
```

Comment:
let timerId = 0 - setInterval return a Number.

Reply:
`undefined` is used also by the `lodash` lib :)

Reply:
Still - better use a Number type instead of 'undefined', always.

Reply:
`null` maybe, but never a number as `0` may ba valid in theory 8-)

Reply:
My understanding is that browser will always return a value higher than 0.

Reply:
I changed it ot `null`.